resource/aws_synthetics_canary: fix minimum timeout_in_seconds (#19515)
hashicorp/terraform-provider-aws: .changelog/19515.txt

```release-note:bug

> resource/aws_synthetics_canary: Fix minimum timeout_in_seconds in run_config

Reviewer: ```suggestion
resource/aws_synthetics_canary: Change minimum `timeout_in_seconds` in `run_config` from `60` to `3`
```

Author: Changed to suggested text
